src/entities/Erc1155Token/Erc1155Token.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
```

Comment:
I hate to see these rules..
You can circumvent it by wrapping trait_type in quotes. I'll suggest it below and flag this comment moving forwards.

Reply:
This is intentional, this data comes from `TokenInfo`. So we don't have control over property names.

Reply:
I will add comment to make that clear.